test/primitives/kernels/test_qkernel_primitives.py

```diff
@@ -0,0 +1,508 @@
+# # This code is part of Qiskit.
+# #
+# # (C) Copyright IBM 2021, 2022.
+# #
+# # This code is licensed under the Apache License, Version 2.0. You may
+# # obtain a copy of this license in the LICENSE.txt file in the root directory
+# # of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+# #
+# # Any modifications or derivative works of this code must retain this
+# # copyright notice, and modified files need to carry a notice indicating
+# # that they have been altered from the originals.
+
+# """ Test QuantumKernel """
+
+import functools
+import unittest
+
+from test import QiskitMachineLearningTestCase
+
+import numpy as np
+import qiskit
+from ddt import data, ddt
+from qiskit import BasicAer, QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.circuit.library import ZZFeatureMap
+from qiskit.transpiler import PassManagerConfig
+from qiskit.transpiler.preset_passmanagers import level_1_pass_manager
+from qiskit.utils import QuantumInstance, algorithm_globals, optionals
+from qiskit.primitives import Sampler
+from qiskit.primitives.fidelity import Fidelity
+from sklearn.svm import SVC
+
+from qiskit_machine_learning.primitives.kernels import (
+    QuantumKernel,
+    PseudoKernel,
+    TrainableQuantumKernel,
+)
+
+
+@ddt
+class TestQuantumKernelClassify(QiskitMachineLearningTestCase):
```

Comment:
Usually, a test class should correspond to a class being tested, not a behaviour being tested.